src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -45,7 +38,8 @@ protected ActionResult execute() {
         gateKeeper.verifyAccessible(
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
-        FeedbackSessionAttributes fs = extractFeedbackSessionData();
+        InstructorFeedbackAddAction newSession = new InstructorFeedbackAddAction();
+        FeedbackSessionAttributes fs = newSession.extractFeedbackSessionData();
```

Comment:
You need to have a subclass object to call superclass method. Else you need to override the superclass method and have to use `super` which gives error in the build which says **overriding method merely calls super** .